### DISCUSSION TITLE
问问，是我打开方式不对吗，一直在加载，登录不上

### DISCUSSION THREAD

**Discussion by CtrlScrollScroll** [mikumifa/biliTickerBuy#180] [提问专区]
直接改cookies.json也没用，点击这里导入也没用，在购买页拖拽导入也没用，开梯子关梯子都不行
defender已关，无三方杀毒软件
另一位有同样问题的仁兄没传好图片没得到解答，我再来偷摸问一次
![2024-06-28](https://github.com/mikumifa/biliTickerBuy/assets/44940408/a845ba9a-bfdf-44ff-a7a8-c090db62d314)

**Comment by mikumifa** [comment]
给个cookie结构？，我看看你的结构对不对

**Reply by mikumifa** [reply]
明天看看，你现在可以浏览器登录吗

**Reply by mikumifa** [reply]
![image](https://github.com/mikumifa/biliTickerBuy/assets/99951454/2fd1a511-e0c9-4214-9e74-46c3d27ccfed)
yong用你的cookie登录进去了

**Reply by CtrlScrollScroll** [reply]
不行，我这里还是一直在走秒数，好奇怪

**Reply by mikumifa** [reply]
你的控制台有什么报错。。

**Reply by CtrlScrollScroll** [reply]
没有报错，控制台最后一句是分享链接72小时后过期 （捂脸）

**Reply by mikumifa** [reply]
母鸡了，你要是会Python的话就看看打断点看看，导入那一步问题在哪
没报错还真不清楚是什么原因

**Reply by CtrlScrollScroll** [reply]
好的吧 谢谢大佬

**Reply by mikumifa** [reply]
你重新解压一下试试

**Reply by CtrlScrollScroll** [reply]
![image](https://github.com/mikumifa/biliTickerBuy/assets/44940408/5728625c-e9a9-4f66-a5f5-5f4b824a5e74)
好神奇，我花了半小时重新装了个纯净系统，居然可以了。我更倾向于我这个系统用久了有乱七八糟的设置忘记改了，总不能是win11和win10的区别吧（）
